_summaries/wu2021rethinking_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: wu2021rethinking
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: aabayomi
+score: 10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+### What is the core idea?
+
+The authors argues using positive random orthogonal features can estimate softmax showing speed and performance gains over SOTA transformer architecture
+
+### How is it realized (technically)?
+
+[1] Only non-negative kernel scores are useful.
+
+<img width="700px" src="wu2021rethinking_2h.png"/>
+
```

Comment:
Will you be explaining this, while presenting or else you can replace this picture with more content on why non-negative kernel scores are useful. How they have technically reduced the time complexity of attention from O(L^2 * D) to O(Lrd). Also maybe add just the required math expressions.